scripts/install

```diff
@@ -148,15 +153,22 @@ yunohost service add $app --description="Low-code programming for event-driven a
 ynh_script_progression --message="Starting a systemd service..." --weight=2
 
 # Start a systemd service
-ynh_systemd_action --service_name=$app --action="start" --log_path="/var/log/$app/$app.log"
+ynh_systemd_action --service_name=$app --action="start"
 
 #=================================================
 # SETUP SSOWAT
 #=================================================
 ynh_script_progression --message="Configuring permissions..." --weight=1
 
-ynh_permission_update --permission="main" --remove="all_users" --add="$admin"
-ynh_permission_create --permission="ui" --url="/ui" --allowed="$admin"
```

Comment:
Eeeeh, should we add something in the upgrade script to delete this legacy permission ?